app/search/models.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import re
+import sqlalchemy
+import json
+from app.package.models import Package
+from app.profile.models import Publisher
+
+
+class DataPackageQuery(object):
+
+    query = ''
```

Comment:
I wanted the query string be available across the methods. So from controller i can only call final method to get the data.